core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/util/LinkInspectorResourceUtil.java

```diff
@@ -59,6 +52,36 @@ public static void removeResource(String path, ResourceResolver resourceResolver
         }
     }
 
+    public static void createNode(String path, ResourceResolver resourceResolver) {
+        try {
+            Session session = resourceResolver.adaptTo(Session.class);
+            if (session == null) {
+                LOG.warn("Session is null, recreating node is interrupted");
+                return;
+            }
+            JcrUtil.createPath(path, JcrConstants.NT_UNSTRUCTURED, session);
+            session.save();
+        } catch (RepositoryException e) {
+            LOG.error(String.format("Failed to recreate node %s", path), e);
+        }
+    }
+
+    public static void addParamToNode(String path, ResourceResolver resourceResolver, String paramName, Long paramValue) {
```

Comment:
Why `paramValue` is a boxed Long, suddenly?)) Use `long`. You can use `int` allright, this is no problem, and then no need to convert to long in [CsvHelperImpl.java](https://github.com/exadel-inc/etoolbox-link-inspector/pull/28/files?diff=unified&w=0#diff-4de93ea8a9319a21106040b10e20a94df2692dfd3082ae236d9d6687be396a85)

Reply:
Origianly we don't have int type in jcr